f5_openstack_agent/lbaasv2/drivers/bigip/listener_service.py

```diff
@@ -56,7 +56,9 @@ def create_listener(self, service, bigips):
             tls['name'] = vip['name']
             tls['partition'] = vip['partition']
 
+        network_id = service['loadbalancer']['network_id']
         for bigip in bigips:
+            self.service_adapter.get_vlan(vip, bigip, network_id)
```

Comment:
Let me ask the question differently.  The VS is the aggregation point for listeners and members.  This PR locks down the VLAN or tunnel added the VS to restrict traffic on the listener side.  But do we also need to add VLAN or tunnel to the VS for members that live on other networks than the listeners?